src/components/videos.jsx

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from "react";
 import Video from "./video";
 import Data from "../storage/db.json";
+// How does this component know when the JSON file changes??
```

Comment:
It doesn't need to know. As long as there's data inside, the component will generate the list.

Reply:
No, that's not right. After I upload a video, the list does not update. The only reason it *seems* to work is because you are running it in development mode, with hot-loading. So the JSON file changes, and the hot-loading rebuilds the app on-the-fly. If you deploy this app, you'll see what I mean. Or run "yarn build; serve -l 3000 -s build" and test it locally.

Reply:
Just to be clear, as written, this component reads the JSON data once, at build time. The only time the list responds to changing data is when the app gets re-built.

Let's separate the backend from the front-end completely here. So we can't store the database data inside the client src code, which is a clever little trick, but cannot work in any production setting, where the server and the client are running on different environment.

So what we need here is for the successful file upload to trigger change of state in the Videos component.